x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -390,6 +391,13 @@ export class TimeSeriesExplorer extends React.Component {
     this.props.appStateHandler(APP_STATE_ACTION.SET_FORECAST_ID, forecastId);
   };
 
+  displayErrorToastMessages = (error, errorMsg) => {
```

Comment:
Is this message cleared out when changing to a different job? If I switch from one of the metrics jobs that gives an error, to a different job, I still see this error on the page:

![image](https://user-images.githubusercontent.com/7405507/116422650-db3fef00-a837-11eb-85d7-434d8d5fa7bc.png)

Reply:
Fixed here [`7626e51` (#98490)](https://github.com/elastic/kibana/pull/98490/commits/7626e515755502893f9fb776dd6dce4827b229d8)